Manual revert of D16012838

wanchaol opened the pull request:
Differential Revision: D16079809

nairbv commented:
@ailzhang fyi could impact XLA

houseroad reviewed (commented):
This also affects ONNX tests I think

Commit:
Manual revert of D16012838

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/22412

Reviewed By: nairbv, houseroad

Differential Revision: D16079809

fbshipit-source-id: d51554755ec8e7cabaf0b88d7122500dcd53b735
